MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -75,25 +75,41 @@ import Foundation
     ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
     ///   - correlationId: Optional. UUID to correlate this request with the server for debugging.
     ///   - delegate: Delegate that receives callbacks for the Get Access Token flow.
-    @objc public func getAccessToken(forceRefresh: Bool = false, correlationId: UUID? = nil, delegate: CredentialsDelegate) {
-        Task {
-            let controllerResponse = await getAccessTokenInternal(
-                forceRefresh: forceRefresh,
-                correlationId: correlationId,
-                cacheAccessor: cacheAccessor
-            )
+    @objc public func getAccessToken(forceRefresh: Bool = false,
+                                     correlationId: UUID? = nil,
+                                     delegate: CredentialsDelegate) {
+        getAccessTokenInternal(forceRefresh: forceRefresh,
+                               scopes: [],
+                               correlationId: correlationId,
+                               delegate: delegate)
+    }
 
-            let delegateDispatcher = CredentialsDelegateDispatcher(delegate: delegate, telemetryUpdate: controllerResponse.telemetryUpdate)
+    /// Retrieves an access token for the account.
```

Comment:
also for this method, can you please align with MSAL Android?

Reply:
Done.